scripts/gen-go-std-tables.sh

```diff
@@ -53,4 +53,8 @@ $(while read path name; do
 	echo "\"${path}.${name}\": true,"
 done <<<"${compiler_intrinsics_table}")
 }
+
+var reflectSkipPkg = map[string]bool{
+	"fmt": true,
+}
```

Comment:
why code generate this at all, if it's a static map? in fact, why have a map when you can just do `path == "fmt"` :)

we might want to expand the logic to also consider other reflect APIs as "harmless" by default, but I don't think we should worry about that for now. I don't think we'll need anything more than just fmt for the time being.

Reply:
I like having everything which can't be obfuscated at the same place